MapboxAndroidWearDemo/src/main/java/com/mapbox/mapboxandroiddemo/analytics/AnalyticsTracker.java

```diff
@@ -0,0 +1,177 @@
+package com.mapbox.mapboxandroiddemo.analytics;
+
+import android.os.Build;
+import android.support.annotation.NonNull;
+
+import com.segment.analytics.Analytics;
+import com.segment.analytics.messages.IdentifyMessage;
+import com.segment.analytics.messages.ScreenMessage;
+import com.segment.analytics.messages.TrackMessage;
+
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+
+/**
+ * This class abstracts various analytics calls to Segment analytics' Java library.
+ */
+
+public class AnalyticsTracker {
+
+  private static volatile AnalyticsTracker analyticsInstance;
+  private Analytics analytics = Analytics.builder("W72jV8mXRPrOkNPFh8Z9tfarSDTYIof4").build();
```

Comment:
So weird that that line still lives on? I've been using .builder() in the constructor for a while now. That exposed key you see above has been rotated and the key is no longer public:  

```java
analytics = Analytics.builder(context.getString(R.string.mapbox_segment_write_key)).build();
```